syntax = "proto3";

package userservice;

service UserService {
  rpc CreateUser(UserRequest) returns (UserResponse);
}

message UserRequest {
  string username = 1;
  string email = 2;
  string password = 3;
}

message UserResponse {
  string user_id = 1;
  string username = 2;
}
